app/models/appointment.rb

```diff
@@ -0,0 +1,10 @@
+class Appointment < ApplicationRecord
+  belongs_to :slot
+  belongs_to :user
+
+  enum interaction: [:online, :face_to_face]
+  after_initialize :set_default_interaction, :if => :new_record?
```

Comment:
maybe this can work with default in the migration file instead of a model method?